src/osemgrep/targeting/Ftree.mli

```diff
@@ -0,0 +1,81 @@
+(*
+   Lazy file tree used for path filtering.
```

Comment:
Maybe explain the name of the file, maybe something like
"Ftree.ml is a tree of FPath.ml"